libs/dh/market-participant/feature-market-participant/src/components/market-participants.component.html

```diff
@@ -14,17 +14,52 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<vater-flex fill="vertical" scrollable *transloco="let t; read: 'marketParticipant.actorsOverview'">
+<watt-data-table
+  vater
+  inset="ml"
+  *transloco="let t; read: 'marketParticipant.actorsOverview'"
+  [searchLabel]="'shared.search' | transloco"
+  [error]="dataSource.error"
+  [ready]="dataSource.called"
+>
+  <h3>{{ t("actors") }}</h3>
+
+  <watt-data-actions>
+    <watt-button icon="download" variant="text" (click)="download()">{{
+      "shared.download" | transloco
+    }}</watt-button>
+
+    <ng-container *dhPermissionRequired="['actors:manage']">
+      <watt-button variant="secondary" (click)="createMarketParticipant()">{{
+        t("create")
+      }}</watt-button>
+    </ng-container>
+
+    <ng-container *dhPermissionRequired="['actors:manage']">
+      <watt-button variant="icon" icon="moreVertical" [matMenuTriggerFor]="menu" />
+
+      <mat-menu #menu="matMenu">
+        <button type="button" mat-menu-item (click)="mergeMarketParticipants()">
+          {{ t("mergeMarketParticipantsButton") }}
+        </button>
+      </mat-menu>
+    </ng-container>
+  </watt-data-actions>
+
+  <watt-data-filters>
+    <dh-market-participants-filters (filter)="fetch($event)" />
+  </watt-data-filters>
+
   <watt-table
-    [dataSource]="tableDataSource()"
+    *transloco="let resolveHeader; read: 'marketParticipant.actorsOverview.columns'"
```

Comment:
nipick: let's use `prefix

`
```suggestion
    *transloco="let resolveHeader; prefix: 'marketParticipant.actorsOverview.columns'"
```